realm/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -326,6 +334,11 @@ public Builder deleteRealmIfMigrationNeeded() {
             return this;
         }
 
+        public Builder durability(SharedGroup.Durability durability) {
```

Comment:
I agree. If we need durability as a parameter it should be moved to a public enum. I am not a big fan of calling it ```durability``` though. The only reason we would need that if we at some point are going to support the ```Durability.ASYNC``` which I am not sure we are going to?

If not I would much prefer this to be called ``inMemory()```